docker/Dockerfile

```diff
@@ -149,16 +155,21 @@ RUN echo "Building Avalon Common Python\n" \
 
 #Build common/crypto_utils intermediate docker image
 FROM python_image as common_crypto_utils_image
+ARG DISTRO
+COPY ./scripts/install_packages /project/avalon/scripts/
 
-RUN apt-get update \
- && apt-get install -y -q \
-    swig
+RUN packages="swig"; \
+    if [ "$DISTRO" = "centos" ] ; then \
+      packages="$packages make gcc gcc-c++"; \
```

Comment:
Looks like  `sudo dnf group install "Development Tools"`  in centos is equivalent to `sudo apt-get install build-essential` in ubuntu